app/controllers/Application.java

```diff
@@ -430,6 +430,7 @@ public static Result flowHistory() {
     if (results.size() == 0) {
       return notFound("Unable to find record on flow url: " + flowDefId);
     }
+    IdUrlPair flowDefPair = new IdUrlPair(flowDefId, results.get(0).flowDefUrl);
```

Comment:
Where should a comment like that go? There are many places where there is UI that has this requirement so I can't really think of a centralized location.